nailgun/entities.py

```diff
@@ -1579,6 +1632,11 @@ def create_missing(self):
                  |-> domain
                  '-> environment
 
+        In case, you have full set of instance attributes that is necessary to
+        pass, don't forget to put False value for your 'create()' method call,
+        so flow will not enter current 'create_missing' method and not throw
+        an exception
+
```

Comment:
This is inaccurate. [The documentation](http://nailgun.readthedocs.org/en/latest/api/nailgun.entity_mixins.html#nailgun.entity_mixins.EntityCreateMixin) gives this brief description of `create_missing`:

> Populate required fields with random values. Required fields that already have a value are not populated. This method is not called by default.

Calling `.create(False)` is only necessary in Robottelo because Robottelo goes out of its way to change NailGun's default behaviour. [`robottelo/__init__.py`](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/__init__.py) contains a call to [`configure_nailgun()`](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/helpers.py#L282). What does that method do? Among other things, it enables automatic dependency creation:

```python
entity_mixins.CREATE_MISSING = True
```